tdishr/TdiDecompileR.c

```diff
@@ -127,11 +126,10 @@ STATIC_CONSTANT char *bname[] = {
   "Opaque",			/*217 */
 };
 
-STATIC_ROUTINE int Append(char *pstr, struct descriptor_d *pd)
-{
+static int Append(const char *const pstr, struct descriptor_d *pd) {
```

Comment:
coding conventions ?

Reply:
should be lowercase but i did not process every file